scripts-available/CDB_OverviewsSupport.sql

```diff
@@ -0,0 +1,173 @@
+-- Auxiliary overviews FUNCTIONS
```

Comment:
mmm, I stashed all my messy commits together and now the actual fix isn't clear at all.
But it's not a rename, it's just some functions from Overviews that are or could be used from other files have been extracted to a separate file.